solidity/dashboard/src/components/LiquidityRewardCard.jsx

```diff
@@ -29,20 +30,28 @@ const LiquidityRewardCard = ({
   withdrawLiquidityRewards,
 }) => {
   const formattedApy = useMemo(() => {
-    const bn = new BigNumber(apy)
-    if (bn.isEqualTo(Infinity)) return <span>&#8734;</span>
-
-    return bn.isLessThan(0.01) && bn.isGreaterThan(0)
-      ? "<0.01%"
-      : bn.multipliedBy(100).decimalPlaces(2, BigNumber.ROUND_DOWN).toString() +
-          "%"
+    const bn = new BigNumber(apy).multipliedBy(100)
+    if (bn.isEqualTo(Infinity)) {
+      return { value: Infinity, prefix: "" }
+    } else if (bn.isLessThan(0.01) && bn.isGreaterThan(0)) {
+      return { value: 0.01, prefix: ">" }
+    } else if (bn.isGreaterThan(999)) {
+      return { value: 999, prefix: "<" }
```

Comment:
I think you mixed up the prefixes for 0.01 and 999. When the APY is lower that 0.01, then we should display it as "<0.01%" which translates to lower than 0.01% (x<0.01 where x is the APY). Similar situation with 999 prefix.